docs/preset-env.md

```diff
@@ -229,40 +229,45 @@ This option is useful for "blacklisting" a transform like `@babel/plugin-transfo
 
 `"usage"` | `"entry"` | `false`, defaults to `false`.
 
-This option configures how `@babel/preset-env` handles polyfills.
+This option configures how `@babel/preset-env` imports various polyfills in your generated code.
 
-When either the `usage` or `entry` options are used, `@babel-preset-env` will add direct references to `core-js` modules as bare imports (or requires). This means `core-js` will be resolved relative to the file itself and needs to be accessible.
+Specifically, this option supports the following external polyfill packages:
 
-Since `@babel/polyfill` was deprecated in 7.4.0, we recommend directly adding `core-js` and setting the version via the [`corejs`](#corejs) option.
```

Comment:
This note is useful for many users, so I don't think that remove it and add a stripped version is a good idea.

Reply:
I have a line below that reads:

```md
> NOTE: Previously, these would be installed via `@babel/polyfill`, which is now deprecated. The above method of installing these polyfill packages is the recommended way.
```

I don't see this as any more stripped down than the line you commented on, just moved down to be more contextual. Do you want me to move this line up?